executor/index_lookup_join.go

```diff
@@ -68,12 +68,16 @@ type IndexLookUpJoin struct {
 	innerPtrBytes [][]byte
 
 	memTracker *memory.Tracker // track memory usage.
+
+	joinResultCh      chan *indexLookUpResult
+	joinChkResourceCh []chan *chunk.Chunk
```

Comment:
It looks like the above two properties are not used in `IndexLookUpJoin`, pls move them to `IndexLookUpHashJoin`.

Reply:
done